src/css/_cards.scss

```diff
@@ -4,6 +4,10 @@ h3 {
   height: 17%;
   margin: 5px 5px 10px 0px;
   transition: .8s ease;
+  &:hover, &:focus {
+    text-shadow: 0px 0px 4px $orange;
+    transition: .8s ease;
+  }
 }
```

Comment:
I wonder if this &: hover is working with the comma and the focus attached or if the focus will have to be its own section?